<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyMT5</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>주문</string>
    </property>
    <addaction name="menu_order_send"/>
    <addaction name="menu_order_total"/>
    <addaction name="menu_order_get"/>
    <addaction name="menu_order_calc_margin"/>
    <addaction name="menu_order_calc_profit"/>
    <addaction name="menu_order_check"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>포지션</string>
    </property>
    <addaction name="menu_position_total"/>
    <addaction name="menu_position_get"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>히스토리</string>
    </property>
    <addaction name="menu_history_orders_total"/>
    <addaction name="menu_history_orders_get"/>
    <addaction name="menu_history_deals_total"/>
    <addaction name="menu_history_deals_get"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_order_send">
   <property name="text">
    <string>order_send</string>
   </property>
  </action>
  <action name="menu_order_total">
   <property name="text">
    <string>order_total</string>
   </property>
  </action>
  <action name="menu_order_get">
   <property name="text">
    <string>order_get</string>
   </property>
  </action>
  <action name="menu_order_calc_margin">
   <property name="text">
    <string>order_calc_margin</string>
   </property>
  </action>
  <action name="menu_order_calc_profit">
   <property name="text">
    <string>order_calc_profit</string>
   </property>
  </action>
  <action name="menu_order_check">
   <property name="text">
    <string>order_check</string>
   </property>
  </action>
  <action name="menu_position_total">
   <property name="text">
    <string>position_total</string>
   </property>
  </action>
  <action name="menu_position_get">
   <property name="text">
    <string>position_get</string>
   </property>
  </action>
  <action name="menu_history_orders_total">
   <property name="text">
    <string>history_orders_total</string>
   </property>
  </action>
  <action name="menu_history_orders_get">
   <property name="text">
    <string>history_orders_get</string>
   </property>
  </action>
  <action name="menu_history_deals_total">
   <property name="text">
    <string>history_deals_total</string>
   </property>
  </action>
  <action name="menu_history_deals_get">
   <property name="text">
    <string>history_deals_get</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
